src_assets/common/assets/web/public/assets/locale/en.json

```diff
@@ -252,7 +252,7 @@
     "origin_web_ui_allowed_lan": "Only those in LAN may access Web UI",
     "origin_web_ui_allowed_pc": "Only localhost may access Web UI",
     "origin_web_ui_allowed_wan": "Anyone may access Web UI",
-    "output_name_desc_unix": "During Sunshine startup, you should see the list of detected displays. Note: You need to use the id value inside the parenthesis.",
+    "output_name_desc_unix": "During Sunshine startup, you should see the list of detected displays. Note: You need to use the id value inside the parenthesis. Below is an example; the actual output can be found in the Troubleshooting tab.",
```

Comment:
@Hazer this locale key is used in a Legacy display vue template, and a New display vue template. Is the "New" display template used at all right now?